flipperdevices/flipperzero-firmware
# [FL-2491] File browser GUI module

# What's new

- File browser GUI module with support for folder navigation
- File browser test app
- File browser dialog module
- File select dialog changed to file browser dialog in all apps
- Unbinding apps from their work folders

# Verification 

- Check Debug Tools -> File Browser test
- Check open/edit/rename/remove functions in all apps

# Checklist (For Reviewer)
